test/navigationLibrary/StandardRoute/RouteSegmentCollection.spec.js

```diff
@@ -53,13 +54,21 @@ ava('.findWaypointsForSegmentName() returns an array of fixes for a given segmen
     t.true(_isEqual(result[0], ['PIRMD', null]), 'Expected to receive and array that is [`{string}`, `{string|null}`]');
 });
 
-ava('.gatherFixNamesForCollection() returns a list of names for each RouteSegmentModel in the collection', t => {
+ava('.gatherFixNamesForCollection() returns a list of entryPoint/exitPoint names for each RouteSegmentModel in the collection', t => {
     const collection = new RouteSegmentCollection(ROUTE_SEGMENTS_MOCK);
     const result = collection.gatherFixNamesForCollection();
 
     t.true(result.length === 8);
 });
 
+ava('.getAllFixNames() returns an array of fix names used by any segment in the collection', (t) => {
+    const collection = new RouteSegmentCollection(ROUTE_SEGMENTS_MOCK);
+    const result = _uniq(collection.getAllFixNames().sort());
```

Comment:
I ask because you are checking the result based on the return of another function. In these instances, usually, I prefer to have an explicit list to compare against because that won't ever change.  When one of those methods changes and the test fails, you now have two vectors of failure instead of just one. 

I know where you're going here, and it's ok, I just want to call out that slight difference.